### PR TITLE
Vector rendering improvements

### PR DESCRIPTION
**Pull request by rochoa**
This includes:
 - Upgrades [tangram-reference](https://github.com/cartodb/tangram-reference): default values and several fixes.
 - [tangram-cartocss](https://github.com/cartodb/tangram-cartocss) from 0.6.0 to 0.9.0:
   * Better handling of symbolizers conditional activation.
   * Better code generation: uses tangram filters and literal values when possible.
   * It enforces default values from reference.
   * Translation of blending property.
   * Better fallback detection support.
   * Removes babel dependencies. The transpiling is handled by webpack: see webpack[.prod].config.js.
   * Adds support for dot-* properties.
   * Not a feature, but worth mentioning: removes a lot of dependencies, avoids the usage of Ramda, and upgrades the yamljs dependency.
 - [tangram.cartodb](https://github.com/cartodb/tangram.cartodb)
   * Uses the new tangram-cartocss version/API.
 - Uses a yet-to-release version of [Tangram which includes an important fix for alpha/opacity](https://github.com/tangrams/tangram/pull/583). This will become, most likely, Tangram 0.14, we will use that version once it's published.

Huge kudos to @dv343 for helping with all the code in this one. 👍

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -3081,9 +2633,9 @@
       "resolved": "https://registry.npmjs.org/y18n/-/y18n-3.2.1.tgz"
     },
     "yamljs": {
-      "version": "0.2.10",
-      "from": "yamljs@>=0.2.8 <0.3.0",
-      "resolved": "https://registry.npmjs.org/yamljs/-/yamljs-0.2.10.tgz",
+      "version": "0.3.0",
```
Comment (nobuti) [inline npm-shrinkwrap.json]: ❤️

**Comment by rochoa** [comment]
What do we have to do with the failing `NEWS.md not updated` check?

**Comment by nobuti** [comment]
If you think it's worthy, there is an enhancement block in the `NEWS.md` file to report this change. If not, just ignore it. I feel it should be in there though.

**Comment by alonsogarciapablo** [comment]
@rochoa can this be merged / release? are we waiting for something?

**Comment by rochoa** [comment]
We were evaluating if we were fixing another bug. We already took the decision to fix it as a separate issue/release. So we will merge this and release it.

**Comment by alonsogarciapablo** [comment]
👏 👏 👏

**Pull request merged**